CHANGELOG.md

```diff
@@ -17,12 +17,23 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+* Pass through facts from Lieutenant API to `parameters.facts` ([#192])
 * Slightly improved error message in case `commodore component new` is called in
   a folder that is not a populated (compiled) catalog ([#183])
 * Vault default values ([#176])
 * Improved `component new` documentation ([#182])
 * `component new` restricts allowed component slugs ([#189])
 
+### Deprecated
+
+* The following parameters will be removed in a future release.
+  They are replace with corresponding values within `parameters.facts` ([#192]).
+  * `parameters.cluster.dist` → `parameters.facts.distribution`
+  * `parameters.cluster.provider` → `parameters.facts.provider`
+  * `parameters.cluster.region` → `parameters.facts.region`
+  * `parameters.cluster.region` → `parameters.facts.region`
```

Comment:
duplicate entry? also do we really want magic UTF-8 in our CHANGELOG.md?

Reply:
I fixed the duplicate. Since the file is UTF-8 encoded, I do not see a reason not to use those characters.